readme.md

```diff
@@ -61,6 +61,12 @@ Client side will be created in `React` using `Javascript` language.
 git clone repo
 ```
 
+### Navigate to project folder
+
+```sh
+cd /path/to/project
```

Comment:
trivial, at least add the name of the folder for the proejct, i smell it should be express-notes-app